components/src/main/java/com/mercadopago/sdk/android/components/Label.kt

```diff
@@ -0,0 +1,437 @@
+package com.mercadopago.sdk.android.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mercadopago.sdk.android.foundation.theme.MercadoPagoTheme
+
+private const val TEXT_GROUP = "Text"
+
+/**
+ * Label Type enum class, used to determine the label type
+ * This its used to change the token color
+ */
+enum class LabelColorType {
+    /**
+     *  Primary: Label of the Primary type
+     */
+    Primary,
+
+    /**
+     *  Secondary: Label of the Secondary type
+     */
+    Secondary,
+
+    /**
+     *  Accent: Label of the Accent type
+     */
+    Accent,
+
+    /**
+     *  Negative: Label of the Negative type
+     */
+    Negative,
+
+    /**
+     *  Inverted: Label of the Inverted type
+     */
+    Inverted
+}
+
+/**
+ * Label Text type enum class, used to determine the label type
+ * This its used to change the text style
+ */
+enum class LabelTextType {
+    /**
+     * Tittle: Label of the Tittle type
+     */
+    Tittle,
+
+    /**
+     * BodyMediumSemiBold: Label of the BodyMediumSemiBold type
+     */
+    BodyMediumSemiBold,
+
+    /**
+     * BodyMediumRegular: Label of the BodyMediumRegular type
+     */
+    BodyMediumRegular,
+
+    /**
+     * BodySmallSemiBold: Label of the BodySmallSemiBold type
+     */
+    BodySmallSemiBold,
+
+    /**
+     * BodySmallRegular: Label of the BodySmallRegular type
+     */
+    BodySmallRegular,
+
+    /**
+     * BodyExtraSmallSemiBold: Label of the BodyExtraSmallSemiBold type
+     */
+    BodyExtraSmallSemiBold,
+}
+
+/**
+ * Label component- This handle the text of the components
+ * This component is used to build others components
+ * handling the text implementation
+ *
+ * @param modifier: label modifier
+ * @param text: label text
+ * @param labelTextStyle: text style, must be using
+ * @param labelColorType: type of label color
+ * @param enabled: Boolean indicates if the component its enabled
+ */
+@Composable
+fun Label(
+    modifier: Modifier = Modifier,
+    text: String = "",
+    labelTextStyle: LabelTextType = LabelTextType.Tittle,
+    labelColorType: LabelColorType = LabelColorType.Primary,
+    enabled: Boolean = true,
+) {
```

Comment:
since the text would be required for this component to exist, this should be the first parameter and required. the same applies for the other styles since we would not always have a title text style.  
```suggestion
@Composable
fun Label(
    text: String,
    labelTextStyle: LabelTextType,
    modifier: Modifier = Modifier,
    labelColorType: LabelColorType = LabelColorType.Primary,
    enabled: Boolean = true,
) {
```